aries-site/src/examples/components/anchor/AnchorExample.js

```diff
@@ -6,11 +6,7 @@ export const AnchorExample = () => {
     <Box direction="column" align="start" gap="medium">
       <Box direction="row" gap="small">
```

Comment:
Is this Box needed? Can we do 
```
<Text>Default anchor:{' '}<Anchor href="#">Anchor</Anchor></Text> 
```
instead?

Reply:
Yep!